tools/git/git-configuration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Configuring Git
+subsection: git
+order: 1
```

Comment:
I think this can't be `order: 1` as other page  has the same order.

Reply:
I missed, thank you and fixed.